meta_definitions/__init__.py

```diff
@@ -23,14 +24,14 @@ def get_all_metrics_from_file(metrics_file):
     ]
```

Comment:
line 22 will need to change to  
if meta_variable.startswith("AREA_") or meta_variable.startswith("DEVICE_")

Reply:
Isn't this for the other files?  I didn't change those.

Reply:
ah, you are right